scaling/src/test/java/io/quarkus/ts/openshift/scaling/ScalingOpenShiftIT.java

```diff
@@ -37,92 +45,94 @@ public void scaleBack() {
      * * Wait for their readiness and verify that both of the replicas are responding.
      */
     @Test
-    public void scaleUpTest() {
-        given()
-                .when().get("/scaling")
-                .then()
-                .statusCode(OK.getStatusCode());
-
-        openShiftUtil.scale(DC_NAME, 2);
-
-        assertThat(openShiftUtil.countReadyReplicas(DC_NAME)).isEqualTo(2);
-
-        Set<String> uniqueIds = Collections.newSetFromMap(new ConcurrentHashMap<>());
-        await().atMost(1, TimeUnit.MINUTES).untilAsserted(() -> {
-            String uniqueId =
-                    given()
-                            .when().get("/scaling")
-                            .then()
-                            .statusCode(OK.getStatusCode())
-                            .extract().asString();
-            uniqueIds.add(uniqueId);
-
-            assertThat(uniqueIds).hasSize(2);
-        });
+    public void scaleUpTest() throws InterruptedException {
+        int expectedUniqueIdsAmount = EXPECTED_SCALE_UP;
```

Comment:
The name "expectedUniqueIdsAmount" is confused to me. I think a proper name is just REPLICAS.